src/client/src/orangehrmBuzzPlugin/components/PostFilters.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="orangehrm-post-filters">
+    <oxd-button
+      :label="isActiveRecent || !isMobile ? 'Most Recent Posts' : ''"
```

Comment:
1. Need i18n lang strings
2. better refactor to these properties into computed property.